packages/core/src/command/patient-import/commands/create-patient.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import { errorToString, patientCreateResponseSchema } from "@metriport/shared";
+import { out } from "../../../util/log";
+import { capture } from "../../../util/notifications";
+import { Config } from "../../../util/config";
+import { PatientPayload } from "../patient-import";
+
+export async function createPatient({
+  cxId,
+  facilityId,
+  patientPayload,
+}: {
+  cxId: string;
+  facilityId: string;
+  patientPayload: PatientPayload;
+}): Promise<string> {
+  const { log, debug } = out(`PatientImport create patient - cxId ${cxId}`);
+  const api = axios.create({ baseURL: Config.getApiUrl() });
+  const patientUrl = `/internal/patient?cxId=${cxId}&facilityId=${facilityId}`;
+  try {
+    const response = await api.post(patientUrl, patientPayload);
+    if (!response.data) throw new Error(`No body returned from ${patientUrl}`);
+    debug(`${patientUrl} resp: ${JSON.stringify(response.data)}`);
+    return patientCreateResponseSchema.parse(response.data).id;
```

Comment:
For later: return the whole object so its more reusable?

Reply:
Yea, we only need the id for further steps (PD + DQ) but if we extend the pipeline I agree.

Reply:
Will handle in v1.